Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -114,7 +113,7 @@ struct ArgumentCoder<ArrayReferenceTuple<Types...>> {
         if (UNLIKELY(!size))
             return;
 
-        (..., encoder.encodeSpan(std::span(arrayReference.template data<Indices>(), size)));
+        (..., encoder.encodeSpan(unsafeMakeSpan(arrayReference.template data<Indices>(), size)));
```

Comment:
ArrayReferenceTuple has a span() accessor now. Slightly nicer not to use the unsafe function here.